extensions/amp-lightbox-gallery/1.0/amp-lightbox-gallery.js

```diff
@@ -73,6 +75,7 @@ class AmpLightboxGallery extends BaseElement {
     super.afterOpen();
     const scroller = this.element.shadowRoot.querySelector('[part=scroller]');
     this.setAsContainer?.(scroller);
+    triggerAnalyticsEvent(this.element, 'lightboxOpened', dict({}));
```

Comment:
nit: can leave off the empty dict, since it is the default value for the param
```suggestion
    triggerAnalyticsEvent(this.element, 'lightboxOpened');
```

Reply:
Good call, done.